### PR TITLE
feat: Add DOI identifier to research-outputs

### PR DESCRIPTION
**Pull request by peterstarling**
https://trello.com/c/Ct1o4197/1395-2-store-doi-for-each-roms-output

### CODE REVIEW

File: apps/asap-server/test/integration/fetch-users.integration-test.ts [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
-      email: 'john.doe@asap.science',
+      email: randomEmail,
       role: 'Grantee' as Role,
       degree: 'MPH',
     };
 
     await createUser(user);
 
-    const result = await users.fetch({});
+    const result = await users.fetch({
```
Comment (peterstarling) [inline apps/asap-server/test/integration/fetch-users.integration-test.ts:30]: or we could just fetch the user by ID? although then we would have to specify the ID on creation so maybe some other time

**Review by peterstarling — COMMENTED** [review]
i cant really approve my own PR :D but basically i think i like your approach to the integration tests. I have created a ticket to set up basic integration tests for each of our entities but this doesn't mean that yours shouldnt already land on master.

File: apps/asap-server/test/integration/dummy.integration-test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export {};
```
Comment (peterstarling) [inline apps/asap-server/test/integration/dummy.integration-test.ts:1]: you probably want to remove this file?

**Comment by peterstarling** [comment]
@harlan-kohll i think you need to review this :D

**Review by harlan-kohll — APPROVED** [review]

**Pull request merged**